spring-boot-modules/spring-boot-testing/src/test/java/com/baeldung/boot/configurationproperties/BindingPropertiesToBeanMethodsUnitTest.java

```diff
@@ -0,0 +1,34 @@
+package com.baeldung.boot.configurationproperties;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+@ExtendWith(SpringExtension.class)
+@EnableConfigurationProperties(value = ServerConfig.class)
+@ContextConfiguration(classes = ServerConfigFactory.class)
+@TestPropertySource("classpath:server-config-test.properties")
+public class BindingPropertiesToBeanMethodsUnitTest {
+
+	@Autowired
+	private ServerConfigFactory configFactory;
+
+	@Test
+	void givenBeanAnnotatedMethod_whenBindingProperties_thenAllFieldsAreSet() {
+
```

Comment:
Let's remove these empty lines at the start of each test method.

Reply:
Done